diff --git a/connect_to_openvpn_with_token.sh b/connect_to_openvpn_with_token.sh
index f1eb570..ce1a4b7 100755
--- a/connect_to_openvpn_with_token.sh
+++ b/connect_to_openvpn_with_token.sh
@@ -173,7 +173,7 @@ if [ "$PIA_DNS" != true ]; then
   echo If you want to also enable PIA DNS, please start the script
   echo with the env var PIA_DNS=true. Example:
   echo $ OVPN_SERVER_IP=\"$OVPN_SERVER_IP\" OVPN_HOSTNAME=\"$OVPN_HOSTNAME\" \
-    PIA_TOKEN=\"$PIA_TOKEN\" CONNECTION_SETTINGS=\"$CONNECTION_SETTINGS\" \
+    PIA_TOKEN=\"***\" CONNECTION_SETTINGS=\"$CONNECTION_SETTINGS\" \
     PIA_PF=true PIA_DNS=true ./connect_to_openvpn_with_token.sh
 else
   cp openvpn_config/openvpn_up_dnsoverwrite.sh /opt/piavpn-manual/openvpn_up.sh
@@ -236,7 +236,7 @@ To disconnect the VPN, run:
 # This section will stop the script if PIA_PF is not set to "true".
 if [ "$PIA_PF" != true ]; then
   echo If you want to also enable port forwarding, you can start the script:
-  echo -e $ ${GREEN}PIA_TOKEN=$PIA_TOKEN \
+  echo -e $ ${GREEN}PIA_TOKEN=*** \
     PF_GATEWAY=$gateway_ip \
     PF_HOSTNAME=$OVPN_HOSTNAME \
     ./port_forwarding.sh${NC}
@@ -257,7 +257,7 @@ echo
 echo
 
 echo -e "Starting procedure to enable port forwarding by running the following command:
-$ ${GREEN}PIA_TOKEN=$PIA_TOKEN \\
+$ ${GREEN}PIA_TOKEN=*** \\
   PF_GATEWAY=$gateway_ip \\
   PF_HOSTNAME=$OVPN_HOSTNAME \\
   ./port_forwarding.sh${NC}"
diff --git a/connect_to_wireguard_with_token.sh b/connect_to_wireguard_with_token.sh
index bda0ce1..5d5eaff 100755
--- a/connect_to_wireguard_with_token.sh
+++ b/connect_to_wireguard_with_token.sh
@@ -163,7 +163,7 @@ To disconnect the VPN, run:
 # This section will stop the script if PIA_PF is not set to "true".
 if [ "$PIA_PF" != true ]; then
   echo If you want to also enable port forwarding, you can start the script:
-  echo -e $ ${GREEN}PIA_TOKEN=$PIA_TOKEN \
+  echo -e $ ${GREEN}PIA_TOKEN=*** \
     PF_GATEWAY=$WG_SERVER_IP \
     PF_HOSTNAME=$WG_HOSTNAME \
     ./port_forwarding.sh${NC}
@@ -184,7 +184,7 @@ echo
 echo
 
 echo -e "Starting procedure to enable port forwarding by running the following command:
-$ ${GREEN}PIA_TOKEN=$PIA_TOKEN \\
+$ ${GREEN}PIA_TOKEN=*** \\
   PF_GATEWAY=$WG_SERVER_IP \\
   PF_HOSTNAME=$WG_HOSTNAME \\
   ./port_forwarding.sh${NC}"
diff --git a/get_region.sh b/get_region.sh
index a59460d..514b652 100755
--- a/get_region.sh
+++ b/get_region.sh
@@ -227,7 +227,7 @@ if [[ -z "$PIA_TOKEN" ]] || [[ $PIA_TOKEN == "" ]]; then
   export PIA_TOKEN
   rm -f /opt/piavpn-manual/token
 else
-  echo -e "Using existing token ${GREEN}$PIA_TOKEN${NC}."
+  echo -e "Using existing token ${GREEN}***${NC}."
   echo
 fi
 
@@ -236,7 +236,7 @@ if [[ $VPN_PROTOCOL == wireguard ]]; then
   echo The ./get_region.sh script got started with
   echo -e ${GREEN}VPN_PROTOCOL=wireguard${NC}, so we will automatically connect to WireGuard,
   echo by running this command:
-  echo -e $ ${GREEN}PIA_TOKEN=$PIA_TOKEN \\
+  echo -e $ ${GREEN}PIA_TOKEN=*** \\
   echo WG_SERVER_IP=$bestServer_WG_IP WG_HOSTNAME=$bestServer_WG_hostname \\
   echo -e PIA_PF=$PIA_PF ./connect_to_wireguard_with_token.sh${NC}
   echo
@@ -257,7 +257,7 @@ if [[ $VPN_PROTOCOL == openvpn* ]]; then
   echo The ./get_region.sh script got started with
   echo -e ${GREEN}VPN_PROTOCOL=$VPN_PROTOCOL${NC}, so we will automatically
   echo connect to OpenVPN, by running this command:
-  echo -e $ ${GREEN}PIA_PF=$PIA_PF PIA_TOKEN=$PIA_TOKEN \\
+  echo -e $ ${GREEN}PIA_PF=$PIA_PF PIA_TOKEN=*** \\
   echo   OVPN_SERVER_IP=$serverIP \\
   echo   OVPN_HOSTNAME=$serverHostname \\
   echo   CONNECTION_SETTINGS=$VPN_PROTOCOL \\
diff --git a/get_token.sh b/get_token.sh
index 6a74935..de42ed6 100755
--- a/get_token.sh
+++ b/get_token.sh
@@ -87,8 +87,8 @@ echo -e ${GREEN}OK!
 echo
 token=$(echo "$generateTokenResponse" | jq -r '.token')
 tokenExpiration=$(timeout_timestamp)
-echo -e PIA_TOKEN=$token${NC}
 echo $token > /opt/piavpn-manual/token || exit 1
+echo -e PIA_TOKEN=***${NC}
 echo $tokenExpiration >> /opt/piavpn-manual/token
 echo 
 echo This token will expire in 24 hours, on $tokenExpiration.
